terminal/src/io.rs

```diff
@@ -310,6 +302,47 @@ pub fn secret_input_with_confirmation() -> SecUtf8 {
     )
 }
 
+pub fn secret_stdin() -> Result<SecUtf8, anyhow::Error> {
+    let mut input: Zeroizing<String> = Zeroizing::new(Default::default());
+    std::io::stdin().read_line(&mut input)?;
```

Comment:
Can't `stdin().read_line()` read into `SecUtf8` type directly, bypassing `Zeroizing`?  `SecUtf8` does everything `Zeroizing` does and more, so maybe we could ditch the `zeroize` dependency altogether?

Reply:
Mmh, it seems that it should work with the latest version of `secstr`: https://docs.rs/secstr/latest/secstr/struct.SecUtf8.html#method.unsecure_mut. 

Unfortunately, we're on 0.3.2 though and it's not supported it that version.

Reply:
Opened a PR in `radicle-keystore` that updates secstr to 0.5.x: https://github.com/radicle-dev/radicle-keystore/pull/35

Reply:
Great, thanks!  I Didn't realise it's going to require a version update 😬

Reply:
Cool, we can fix this in a later PR

Reply:
Tracked here: https://github.com/radicle-dev/radicle-cli/issues/240